src/components/windows/types/agenda/editAgendaEntry.jsx

```diff
@@ -0,0 +1,166 @@
+import { useForm } from "react-hook-form";
+import { InnerContainer, InnerContainerRow, InputButton, InputCheckbox, InputContainer, InputElement, InputLabel, LabelWarning, PanelButton } from "../../../dashboard"
+import { Agenda, getCurrentEvent } from '@phoenixlan/phoenix.js'
+import { FormButton, FormInput } from "../../../form";
+import { useContext, useEffect, useState } from "react";
+import { Notice } from "../../../containers/notice";
+import { AuthenticationContext } from "../../../authentication";
+
+export const EditAgendaEntry = ({functions, entries}) => {
+    const { register, handleSubmit, watch, formState: { errors } } = useForm();
+
+    const [ title, setTitle ]											= useState(entries.title);
+    const [ description, setDescription ]								= useState(entries.description);
+    const [ time, setTime]												= useState(entries.time ? new Date(entries.time ? entries.time*1000+7200000 : undefined).toISOString().slice(0, 16) : undefined)
```

Comment:
Would be nice with a comment here to explain what the magical number means. Also, I recommend not using `slice` for time strings. Before you know it you are slicing something of varying length - better to use `split` or similar where you can split on syntactical elements.

Reply:
I see what you mean, but when it comes to this variable, the time variable is built up like this `2023-11-23T22:11:00.000Z`. At which point will will this variable change length? 

I am slicing everything after the `^`, when will the length before this change?
```
2023-11-23T22:11:00.000Z
                ^
```